server/src/main/scala/org/http4s/server/middleware/Logger.scala

```diff
@@ -44,7 +43,9 @@ object Logger {
     val bodyStream = if (logBody && isText) {
       message.bodyAsText(charset.getOrElse(Charset.`UTF-8`))
     } else if (logBody) {
-      message.body.map(ByteVector.fromByte).map(_.toHex)
+      message.body
+        .fold(new StringBuilder)((sb, b) => sb.append(String.format("%02X", Int.box(b & 0xff))))
```

Comment:
This `String.format` isn't dreadfully slow?

Reply:
I think it is. Good catch. I can use a better encoder